src/command/expire/expire.c

```diff
@@ -968,7 +1050,8 @@ cmdExpire(void)
 
                 // Remove all files on disk that are now expired
                 removeExpiredBackup(infoBackup, adhocBackupLabel, repoIdx);
-                removeExpiredArchive(infoBackup, timeBasedFullRetention, repoIdx);
+                removeExpiredArchive(infoBackup, timeBasedFullRetention, repoIdx);               
```

Comment:
A bunch of trailing spaces were added here, please remove. It's better if your editor does this automatically.

Reply:
Oh, indeed, sorry.
(Just added trim_trailing_white_space_on_save and ensure_newline_at_eof_on_save to my sublimeText3 config)
Fixed in 6d2b9f5.

Reply:
I'm surprised that Sublime Text is not honoring the project `.editorconfig` file. Maybe a setting?

Reply:
I've tried to add a specific plugins for that but it doesn't seem to load the `.editorconfig`. I'll have to investigate further on this later but at least the trim option is there now ;-)